sweep/sweeper.go

```diff
@@ -390,10 +402,11 @@ func (s *UtxoSweeper) SweepInput(input input.Input,
 	}
 
 	log.Infof("Sweep request received: out_point=%v, witness_type=%v, "+
-		"time_lock=%v, amount=%v, fee_preference=%v", input.OutPoint(),
-		input.WitnessType(), input.BlocksToMaturity(),
+		"time_lock=%v, amount=%v, fee_preference=%v, "+
+		"force=%v",
```

Comment:
Nit: the args can continue on this line.

Reply:
fixed